Tiltfile

```diff
@@ -11,6 +11,7 @@ include_if_exists('../daily-api/Tiltfile')
 include_if_exists('../heimdall/Tiltfile')
 include_if_exists('../post-scraper-one-ai/Tiltfile')
 include_if_exists('../njord/Tiltfile')
+include_if_exists('../freyja/Tiltfile')
```

Comment:
Most engineers will not need it, so I'm not sure if we need to add it for everyone. it will bloat the system